### DISCUSSION TITLE
Is there any support of Session History screen?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [flyerhq/flutter_chat_ui#115] [Ideas & Requests]
So many thanks for this looks wonderful lib, haven't try it but from the effect img looks very promising.
Wanna ask if there any support on Session History screen? I mean sessions to conversion detail page. We have conversasion page then how about Sessions?

**Comment by demchenkoalex** [comment]
Hey @jinfagang we specifically did not include sessions screens, because this conversation UI can be used in different scenarios, like chat with a support page, just a random conversation (not saved in a session) or whatever chat use cases might be. Also, sessions pages are quite different for everyone, some may have stories on top, some may have custom user bubbles designs etc etc, so you have the conversation UI and then you can use it for everything you want, but you need to write some code of course, including sessions UI in your case. See the example in our [Firebase package](https://github.com/flyerhq/flutter_firebase_chat_core), `rooms` are done outside the chat UI.

**Comment by lucasjinreal** [comment]
@demchenkoalex  thanks for your reply. 

Yes, sessions can be very customizable, but it is essential for lib provides a standard way to orgnise the data model, I mean, at least a simple enough exmple to provide how to build upon it. Did u guys planned to such an helpful feature for users?

**Comment by demchenkoalex** [comment]
As I said in the previous message, example can be found in the Firebase library, specifically [this](https://github.com/flyerhq/flutter_firebase_chat_core/blob/main/example/lib/rooms.dart) file, which is a `ListView` with rows that contain an image and a text. I mean seriously, why would I add this to the package? It's like 40 lines of code without a design 🙂 and if you are meaning something "more advanced", we go back to the part where we agreed that sessions are very customizable, and given that we already have tons of ifs in a conversation UI to handle all customization people want, we will just not handle another big chunk, just not enough value in this.

**Reply by lucasjinreal** [reply]
Muchas gracias.
